data.sql

```diff
@@ -1,21 +1,56 @@
 /* Populate database with sample data. */
 
+---------------------------------- fourth part  ----------------------------------
```

Comment:
| Should be Removed |
|-|
| ![image](https://github.com/MarcoDiaz2000/vet-clinic-database/assets/93039359/fad8d653-859b-4ca8-95cf-f6e3134fc2b7) |
| ![image](https://github.com/MarcoDiaz2000/vet-clinic-database/assets/93039359/b63191fa-17cc-40c9-85a0-756357512c0a) |
| ![image](https://github.com/MarcoDiaz2000/vet-clinic-database/assets/93039359/6547aafd-78a0-4900-9d37-4f41cbadc9f9) |
| ![image](https://github.com/MarcoDiaz2000/vet-clinic-database/assets/93039359/3de59c56-668d-4201-890d-70416875a763) |
| ![image](https://github.com/MarcoDiaz2000/vet-clinic-database/assets/93039359/7da6f7da-2053-424d-afec-08c212bf8b62) |
| ![image](https://github.com/MarcoDiaz2000/vet-clinic-database/assets/93039359/b17564f0-9157-493f-be10-3dc64019b8f4) |

- [ ] It would be nice if you remove this unnecessary information from your readme file to make your readme more descriptive and professional 👍